### PR TITLE
Subscription settings added

### PR DESCRIPTION
**Pull request by marslanabdulrauf**
Description:

This PR adds Subscription Setting for LMS and CMS

JIRA:
https://edlyio.atlassian.net/browse/EDLY-5008

### CODE REVIEW

**Review by faatehsultan — APPROVED** [review]

File: lms/envs/common.py [hunk 1]
```diff
@@ -3998,3 +4010,37 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
...
+PLAN_FEATURES = {
+    TRIAL: {
+        ADDITIONAL_USER_PRICE: 0,
+        MONTHLY_ACTIVE_USERS: 20,
+        NUMBER_OF_REGISTERED_USERS: 20,
+        NUMBER_OF_COURSES: 3,
+        STAFF_USERS: 1,
+        WP_ADMIN_USERS: 1,
+        COURSE_AUTHERS: 1,
+
```
Comment (taimoor-ahmed-1) [inline lms/envs/common.py:4024]: kindly remove blank line

File: openedx/features/edly/constants.py [hunk 2]
```diff
@@ -8,3 +8,12 @@
...
 DEACTIVATED = 'deactivated'
+
+# FEATURE FLAGS
+ADDITIONAL_USER_PRICE = 'additional_user_price'
+MONTHLY_ACTIVE_USERS = 'monthly_active_users'
+NUMBER_OF_REGISTERED_USERS = 'number_of_registered_users'
+NUMBER_OF_COURSES = 'number_of_courses'
+STAFF_USERS = 'staff_users'
+WP_ADMIN_USERS = 'wp_admin_users'
+COURSE_AUTHERS = 'course_authers'
```
Comment (madnihamza1841) [inline openedx/features/edly/constants.py:19]: Kindly correct the spellings for authors.

Comment (madnihamza1841) [reply openedx/features/edly/constants.py:19]: COURSE_AUTHOURS and course_authors

File: lms/envs/common.py [hunk 3]
```diff
@@ -3998,3 +4010,37 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
...
+    },
+    ELITE: {
+        ADDITIONAL_USER_PRICE: 2,
+        MONTHLY_ACTIVE_USERS: 5000,
+        NUMBER_OF_REGISTERED_USERS: 50000,
+        NUMBER_OF_COURSES: 200,
+        STAFF_USERS: 3,
+        WP_ADMIN_USERS: 3,
+        COURSE_AUTHERS: 10,
```
Comment (madnihamza1841) [inline lms/envs/common.py:4043]: Correct the spelling here too for COURSE_AUTHOURS

File: lms/envs/common.py [hunk 4]
```diff
@@ -3998,3 +4010,37 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
...
+
+    },
+    ESSENTIALS: {
+        ADDITIONAL_USER_PRICE: 2,
+        MONTHLY_ACTIVE_USERS: 1000,
+        NUMBER_OF_REGISTERED_USERS: 5000,
+        NUMBER_OF_COURSES: 50,
+        STAFF_USERS: 1,
+        WP_ADMIN_USERS: 1,
+        COURSE_AUTHERS: 3,
```
Comment (madnihamza1841) [inline lms/envs/common.py:4033]: COURSE_AUTHOURS spelling

File: lms/envs/common.py [hunk 5]
```diff
@@ -3998,3 +4010,37 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
...
+# Subscription Plans
+PLAN_FEATURES = {
+    TRIAL: {
+        ADDITIONAL_USER_PRICE: 0,
+        MONTHLY_ACTIVE_USERS: 20,
+        NUMBER_OF_REGISTERED_USERS: 20,
+        NUMBER_OF_COURSES: 3,
+        STAFF_USERS: 1,
+        WP_ADMIN_USERS: 1,
+        COURSE_AUTHERS: 1,
```
Comment (madnihamza1841) [inline lms/envs/common.py:4023]: COURSE_AUTHOURS spelling

File: lms/envs/common.py [hunk 6]
```diff
@@ -52,6 +52,18 @@
 )
 from openedx.core.lib.derived import derived, derived_collection_entry
 from openedx.core.release import doc_version
+from openedx.features.edly.constants import (
+    ADDITIONAL_USER_PRICE,
+    COURSE_AUTHERS,
```
Comment (madnihamza1841) [inline lms/envs/common.py:57]: Spelling mistake

**Review by madnihamza1841 — CHANGES_REQUESTED** [review]

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Pull request merged**